packages/suite/src/services/google.ts

```diff
@@ -264,12 +288,17 @@ class Client {
             const query = new URLSearchParams(apiParams.query as Record<string, string>).toString();
             url += `?${query}`;
         }
+        const accessToken = await this.oauth2Client.getAccessToken();
+
+        if (!accessToken || !accessToken.token) {
+            throw { error: 'no access_token provided' };
```

Comment:
throw Error

Reply:
I am removing it. If something is wrong with token, api will tell us.